test/unit/dredd-command-test.coffee

```diff
@@ -272,6 +272,61 @@ describe "DreddCommand class", () ->
       it 'should save configuration', ->
         assert.isTrue configUtilsStub.save.called
 
+    describe '"init"', ->
```

Comment:
Aren't these tests redundant? I'm not sure how important the language is for this particular test. I think it tests just basic functionality of the `dredd init` command itself.